src/user/room.ts

```diff
@@ -791,29 +884,32 @@ export class Room extends Accessory implements Sayable {
   }
 
   /**
-   * Force reload data for Room
-   * @private
-   * @deprecated use sync() instead
-   * @returns {Promise<void>}
+   * @description
+   * Force reload data for Room, use {@link Room#sync} instead
```

Comment:
should use `@ignore` here.

So does `sync()` because they are not the public API

Reply:
Why we do not make this API public?